sdk/servicebus/Microsoft.Azure.ServiceBus/src/Core/MessageReceiver.cs

```diff
@@ -899,6 +899,30 @@ public void RegisterMessageHandler(Func<Message, CancellationToken, Task> handle
             this.OnMessageHandler(messageHandlerOptions, handler);
         }
 
+        /// <summary>
+        /// Unregister messgae hander from the receiver if there is active message handler registered. 
+        /// </summary>
+        public void UnregisterMessageHandler()
```

Comment:
Not sure if this is an acceptable contract. Should we allow customer to input `Func<ExceptionReceivedEventArgs, Task> exceptionReceivedHandler` as we did in RegisterMessageHandler to notify customer with this exception when exceptions arises in this method?